jerry-core/ecma/builtin-objects/ecma-builtin-dataview-prototype.c

```diff
@@ -193,27 +191,36 @@ ecma_builtin_dataview_prototype_dispatch_routine (uint8_t builtin_routine_id, /*
     case ECMA_DATAVIEW_PROTOTYPE_SET_BIGUINT64:
 #endif /* JERRY_BUILTIN_BIGINT */
     {
-      ecma_value_t value_to_set = arguments_number > 1 ? arguments_list_p[1] : ECMA_VALUE_UNDEFINED;
       ecma_value_t little_endian = arguments_number > 2 ? arguments_list_p[2] : ECMA_VALUE_FALSE;
       ecma_typedarray_type_t id = (ecma_typedarray_type_t) (builtin_routine_id - ECMA_DATAVIEW_PROTOTYPE_SET_INT8);
 
-      return ecma_op_dataview_get_set_view_value (this_arg, byte_offset, little_endian, value_to_set, id);
+      return ecma_op_dataview_get_set_view_value (this_arg,
+                                                  arguments_list_p[0],
+                                                  little_endian,
```

Comment:
Since the undefined and false both results false in toBoolean operation this checks is truly unnecessary. @batizdaniel  please remove this redundant check as well.